File: src/app/model/generated-token.ts
import { Injectable } from "@angular/core";
import { Organization } from "./organization";

@Injectable({
    providedIn: 'root'
  })
  
export class GeneratedToken {
    token:string="";
    organization:Organization=new Organization();
    // constructor(token:string){
    //     this.token=token;
    // }
    public getToken():string{
        return this.token;
    }

    public setToken(token:string){
        this.token=token;
    }

    public getOrganization():Organization{
        return this.organization;
    }

    public setOrganization(organization:Organization){
        this.organization=organization;
    }
}

File: AssetManagementSystem/src/main/java/com/freelancer/assetmanagement/model/GeneratedToken.java
package com.freelancer.assetmanagement.model;

import org.springframework.stereotype.Component;

import com.freelancer.assetmanagement.dto.EmployeeDto;

@Component
public class GeneratedToken {

	private String token;
	private Organization organization;
    private EmployeeDto employeeDto;
	
	
	public EmployeeDto getEmployeeDto() {
		return employeeDto;
	}
 
	public void setEmployeeDto(EmployeeDto employeeDto) {
		this.employeeDto = employeeDto;
	}
	
	public Organization getOrganization() {
		return organization;
	}

	public void setOrganization(Organization organization) {
		this.organization = organization;
	}

	public GeneratedToken() {
		super();
	}

	public GeneratedToken(String token) {
		super();
		this.token = token;
	}

	public String getToken() {
		return token;
	}

	public void setToken(String token) {
		this.token = token;
	}

	@Override
	public String toString() {
		return "GeneratedToken [token=" + token + ", organization=" + organization + "]";
	}

	
}

